source/clear-linux/tutorials/multi-boot/multi-boot.rst

```diff
@@ -5,17 +5,18 @@ Multi-boot Clear Linux with other operating systems
 
 Starting with version 16140, |CLOSIA| uses the Systemd-Boot boot loader,
 which does not support multi-booting without manual manipulation. This
-document shows how to modify the boot loader for |CL| to work with other
-operating systems.
+tutorial shows how to modify the boot loader for |CL| to work with other
+:abbr:`OSes (operating systems)`.
 
 Process overview
 ****************
 
-The general process to install other operating systems for a
-multi-booting computer is as follows:
+The process to install other operating systems for a multi-booting computer is
+described below. Install |CL| first, then install other operating systems in
+any order.
 
 #. Install |CL| first with a EFI partition large enough to store the kernels
-   of other operating systems and their initrds in the case of Linux\*
+   of other operating systems and their initrds in the case of Linux
```

Comment:
I'd like to add a comma after "initrds".  Should be like this: of other operating systems and their initrds, in the case of Linux